<launch>


<arg name="webcam" default="0" />
<arg name="enable_right_hand" default="enable" />
<arg name="enable_left_hand" default="enable" />
<arg name="enable_pose" default="enable" />


<param name="webcam" value="$(arg webcam)"/>
<param name="enable_right_hand"  value="$(arg enable_right_hand)"/>
<param name="enable_left_hand" value="$(arg enable_left_hand)"/>
<param name="enable_pose"  value="$(arg enable_pose)"/>
<node pkg="sk_tracking" type="Publisher.py" name="Publisher" output="screen"> 
 
</node>

</launch>
